testing/e2e/cluster_supply_chain_test.go

```diff
@@ -21,22 +21,56 @@ package integration_test
 
 import (
 	"path/filepath"
+	"regexp"
+	"strings"
 	"testing"
 
+	"github.com/google/go-cmp/cmp"
 	it "github.com/vmware-tanzu/apps-cli-plugin/testing/suite"
 )
 
 func TestClusterSupplyChain(t *testing.T) {
 	testSuite := it.CommandLineIntegrationTestSuite{
 		{
-			Name:                      "List the existing supply chains",
-			Command:                   *it.NewTanzuAppsCommandLine("cluster-supply-chain", "list"),
-			ExpectedCommandLineOutput: it.GetFileAsString(t, filepath.Join(it.ConsoleOutBasePath, "list-csc", "test-list-csc.txt")),
+			Name:    "List the existing supply chains",
+			Command: *it.NewTanzuAppsCommandLine("cluster-supply-chain", "list"),
+			Verify: func(t *testing.T, output string, err error) {
+				match, err := regexp.MatchString(`.*.\nci-test[ ]{3,}[<>a-zA-Z]{5,}[ ]{3,}[0-9]{1,}s\n.*`, output)
+				if !match {
+					t.Error("Expected 'ci-test   <status>   0s' to be present in the output")
+					t.FailNow()
+				}
+				if err != nil {
+					t.Error("Error while validating the output", err)
+					t.FailNow()
+				}
+				const outFoot = `
+To view details: "tanzu apps cluster-supply-chain get <name>"
+
+`
+				if !strings.HasSuffix(output, outFoot) {
+					t.Errorf("Expected %s to be present in the output", outFoot)
+					t.FailNow()
+				}
+
+			},
 		},
 		{
-			Name:                      "Get the existing supply chain",
-			Command:                   *it.NewTanzuAppsCommandLine("cluster-supply-chain", "get", "ci-test"),
-			ExpectedCommandLineOutput: it.GetFileAsString(t, filepath.Join(it.ConsoleOutBasePath, "get-csc", "test-get-ci-test-csc.txt")),
+			Name:    "Get the existing supply chain",
+			Command: *it.NewTanzuAppsCommandLine("cluster-supply-chain", "get", "ci-test"),
+			Verify: func(t *testing.T, output string, err error) {
+				const outHead = "---\n# ci-test:"
```

Comment:
update name from `outHead` to `expectedHeader`

Reply:
Updated